jupyter/trustyai/ubi9-python-3.9/test/test_notebooks.ipynb

```diff
@@ -0,0 +1,90 @@
+{
```

Comment:
nit: can we rename the test notebooks to `test_notebook.ipynb` to keep it consistent with naming of other test notebooks we have.